cellfinder_napari/curation.py

```diff
@@ -525,7 +554,7 @@ def extract_cubes(
         print("Done")
 
 
-def extract_batches(cube_generator, output_directory):
+def extract_batches(cube_generator, output_directory: Path):
```

Comment:
What about `cube_generator`'s type?

Reply:
It's `cellfinder_core.classify.cube_generator.CubeGeneratorFromFile`, but since that is an import within a function at the moment I was wary of putting the import at the top of the file and adding the typing in case there's a good reason why the import is inside the function.